ports/mimxrt/Makefile

```diff
@@ -217,6 +247,13 @@ SRC_C += \
 	ticks.c \
 	tusb_port.c \
 
+ifeq ($(MICROPY_PY_BLUETOOTH),1)
+SRC_C += \
+	drivers/ninaw10/nina_bt_hci.c \
+    mpbthciport.c \
+    mpnimbleport.c
+endif
+
```

Comment:
This is already included somewhere else in the Makefile, I'd add to it:

```C
ifeq ($(MICROPY_PY_BLUETOOTH),1)
SRC_C += mpbthciport.c
ifeq ($(MICROPY_PY_NETWORK_NINAW10),1)
SRC_C += drivers/ninaw10/nina_bt_hci.c
endif
endif # MICROPY_PY_BLUETOOTH
```

Reply:
That's settled now, I hope.